src/ua/hillel/FindWordPosition.java

```diff
@@ -0,0 +1,14 @@
+package ua.hillel;
+
+public class FindWordPosition {
+    public static void main(String[] args) {
+        String source;
```

Comment:
The same here, please extract the code from method main to a separate method, and name it `findWordPosition()`